service/controller/v2/arm_templates/route_table_setup.json

```diff
@@ -27,6 +34,7 @@
     {
       "type": "Microsoft.Network/routeTables",
       "name": "[variables('name')]",
+      "condition": "[equals(parameters('initialProvisioning'), 'Yes')]",
```

Comment:
Disable route table creation if cluster already provisioned.

Unfortunately there is no better way to prevent this. Azure ARM templates are declarative, so if we do not define any routes in ARM template, ARM will wipe existing routes.